### PR TITLE
Rbac users preview

### PR DESCRIPTION
**Pull request by Hyperkid123**
Added user detail preview component.

https://5b322e283813f01ea4c7e59d--confident-banach-ee0a7a.netlify.com/?selectedKind=Rbac%20forms&selectedStory=Rbac%20user%20preview&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

### CODE REVIEW

File: src/rbac-forms/rbacUserPreview.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      </PreviewRow>
+      <PreviewRow label={__('Role')}>
+        {previewLink(user.role.label, 'user', user.role.onClick)}
+      </PreviewRow>
+    </Grid>
+  </div>
+);
+
+// eslint-disable-next-line
+const emailPattern = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (karelhala) [inline src/rbac-forms/rbacUserPreview.jsx]: Can you create new file with some constants and move this there? So we can reuse such specific constants?

Comment (Hyperkid123) [reply src/rbac-forms/rbacUserPreview.jsx]: will do

**Comment by Hyperkid123** [comment]
@karelhala i have created manageiq-validators package, where we should put all non standard validation rules (validators for forms for example).

**Comment by karelhala** [comment]
@Hyperkid123 merge conflicts and we're good to go.

**Comment by Hyperkid123** [comment]
@karelhala done

**Pull request merged**